docs/release/trg-0/trg-1-4.md

```diff
@@ -16,7 +16,7 @@ Also updating the static files will be easier and faster to do for every committ
 
 ## Description
 
-All diagram files **must** be created with code and either stored as `.svg` file within our repositories or implemented in your documentation.
+All diagram files **must** be either created with code languages or stored as `.svg` file within our repositories.
 After a committers [decision](https://github.com/eclipse-tractusx/sig-infra/discussions/19) there are 2 languages accepted, either [PUML](https://plantuml.com/en/) or [Mermaid](https://mermaid.js.org/).
 These diagram code files (`.puml`,`.mmd`,`.mermaid`) and / or static diagram `.svg` files **must** be stored within the `docs/` folder as described in [TRG 2.03](https://eclipse-tractusx.github.io/docs/release/trg-2/trg-2-3#docs).
```

Comment:
Maybe there is a misunderstanding on my side, but the documentation says the svg file must be stored in the `docs` folder, but the example svg file is stored in `static/img/alice-bob-drawio.svg`